recipes/default.rb

```diff
@@ -51,3 +51,10 @@
   user node['timesync']['user']
   action :start_or_graceful_reload
 end
+
+directory '/opt/timesync/source' do
```

Comment:
I created a PR to fix the issue in nodejs-webapp. https://github.com/osuosl-cookbooks/nodejs-webapp/pull/6